OxfordBodleian/Ullendorff/BDLeu4.xml

```diff
@@ -237,25 +271,428 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <extent>
+                                        <measure unit="leaf" quantity="1">1</measure>
+                                        <locus target="#2"/>
+                                        <dimensions type="leaf" unit="mm" xml:id="leafdim2">
+                                            <height>204</height>
+                                            <width>193</width>
+                                        </dimensions>
+                                        <note corresp="#leafdim2">The leaf, which originally was larger, has been trimmed to match the size of the leaves of
+                                            <ref target="#p1">Unit 1</ref>.</note>
+                                    </extent>
+                                </supportDesc>
+                                                                
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="20">
+                                        <locus target="#2"></locus>
+                                        <note>The folio has been erroneously bound, so that the original outer part of the folio is now found
+                                            in the gut. The lower and current outer part of the folio have been trimmed with loss of text.
+                                        Based on the amount of text loss, the leaf must originally have contained an additional six to seven written lines.</note>
+                                        <note>There are 10–12 characters per line.</note>
+<!--                                        <dimensions unit="mm" xml:id="textarea2">
+                                            <height></height>
+                                            <width></width>
+                                        </dimensions>
+                                        <dimensions type="margin" unit="mm" xml:id="margin2">
+                                            <dim type="top"></dim>
+                                            <dim type="bottom">0</dim>
+                                            <dim type="right">0</dim>
+                                            <dim type="left"></dim>
+                                        </dimensions>-->
+                                        <ab type="ruling" subtype="pattern">
+                                            <locus target="#2"/>
+                                            Ruling pattern: 1A-1A-1A1A/0-0/0-0/?.</ab>
+                                        <note>Due to the trimming of the folio, the extent of the ruling lines and the lower end of the vertical lines
+                                            cannot be determined.</note>
+                                        <ab type="pricking">Ruling is visible; the areas expected to contain traces of pricking are
+                                            now hidden inside the quire.</ab>
```

Comment:
can a synonym be found for "hidden"? This could maybe be interpreted as intentional. Are now not accessible due to the tight binding? (or something of the sort)?